recipes/pegasusio/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pegasusio" %}
+{% set version = "0.2.8.post2" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: "95f678bbed68b12da8f05304738b633ff111d9043df2f5753636081350cb5a9b"
+
+
+build:
+    number: 0
+    entry_points:
```

Comment:
Please put the following line in the `build:` section:

    skip: True  # [py27]